### ISSUE TITLE
[Assistive tech and device support] Pages require sensory characteristics to be understood or operable. (11.11.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#28326]
#### VFS team name
Healthcare Experience

#### VFS product name
Check-in MVP

#### Point of Contact/Reviewers
Josh Kim

---

### Platform Issue
Pages require sensory characteristics to be understood or operable.

### Issue Details
Upon successfully checking in, focus is sent to "Not sure where to wait" instead of the confirmation heading.

### Supporting Guidance
[Category 11, Issue 11](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.3.3_A

---

### Platform Recommendation
Focus should go to the `h1` of the confirmation page.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] VFS team closes the ticket when the issue has been resolved

**Comment by jparker0425** [comment]
During Midpoint Review, Accessibility stated focus was good, so we are confirming that nothing changed at this time.

**Comment by mdewey** [comment]
After working with Platform, this should be addressed in staging.

**Comment by shiragoodman** [comment]
adding labels so ticket is consistent with post-pilot process
